language-support/ts/daml-react/README.md

```diff
@@ -47,23 +47,16 @@ Now you can use the following React hooks to interact with a DAML ledger:
 const party = useParty()
 ```
 
-`useExercise`
+`useLedger`
 -------------
-`useExercise` returns a function to exercise a choice by contract id.
+`useLedger` returns an instance of the `Ledger` class of @daml/ledger to interact with the DAML
+ledger.
 
 ```typescript
-const [exerciseChoice] = useExercise(ContractTemplate.ChoiceName)
-const onClick = () => exerciseChoice(contractId, argument)
+const ledger = useLedger()
+const newContract = await ledger.create(ContractTemplate, arguments)
```

Comment:
My impression (from tools like prettier) is that ending statements with semicolons is the recommended way. It definitely simplifies parsing, both for humans and the JS parser.
```suggestion
const ledger = useLedger();
const newContract = await ledger.create(ContractTemplate, arguments);
```

Reply:
Can please also keep an example for `ledger.exercise` in addition to `ledger.create`. That should make the docs more accessible.

Reply:
added examples for archive/exercise.